apps/test/util/runTests.js

```diff
@@ -50,7 +50,10 @@ exec(command, function (err, stdout, stderr) {
 
   var globs = [
     './test/*.js',
+    './test/applab/*.js',
     './test/calc/*.js',
+    './test/craft/*.js',
+    './test/gamelab/*.js',
```

Comment:
:exclamation: :exclamation: :exclamation: How did we miss this?  I remember writing gamelab/reducersTest.js using red-green-refactor... AUGH I iterated by running `mocha test/gamelab/reducersTest.js` because it is so much faster, and not through our testrunner.  :dizzy_face: :palm_tree: